spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JpqlQueryRenderer.java

```diff
@@ -2008,9 +2005,7 @@ public List<JpaQueryParsingToken> visitGeneral_case_expression(JpqlParser.Genera
 
 		tokens.add(new JpaQueryParsingToken(ctx.CASE()));
 
-		ctx.when_clause().forEach(whenClauseContext -> {
-			tokens.addAll(visit(whenClauseContext));
-		});
+		ctx.when_clause().forEach(whenClauseContext -> tokens.addAll(visit(whenClauseContext)));
 
```

Comment:
I am usually a fan of shortening things like this, but since this is an ANTLR Visitor implementation, I actually prefer to retain it the way it is. It makes it easy when grammar tweaks require me to add another `tokens.addAll(...)` inside that clause. And since this pattern is used A LOT in this class (and its counterpart), it becomes easier to maintain if I retain the pattern consistently everywhere.